dev/tests/integration/framework/Magento/TestFramework/Application.php

```diff
@@ -438,7 +438,7 @@ public function cleanup()
      * @return void
      * @throws \Magento\Framework\Exception\LocalizedException
      */
-    public function install()
+    public function install($cleanup)
```

Comment:
Could you please take a look at `Magento\TestFramework\WebApiApplication::install()` this method signature must be compatible with `Magento\TestFramework\Application::install($cleanup)`